<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsGroup</class>
 <widget class="QWidget" name="NewsGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>alt.binaries.example</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/16x16_ico_png/ico_news.png</normaloff>:/resource/16x16_ico_png/ico_news.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpView">
     <property name="title">
      <string>alt.binaries.example</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="chkRefreshAutomatically">
          <property name="text">
           <string>Refresh group automatically on open</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>0</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>16</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>16</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="loader">
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadMore">
        <property name="text">
         <string>Load older headers ...</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>btnLoadMore</zorder>
     <zorder>tableView</zorder>
     <zorder>progressBar</zorder>
     <zorder>loader</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>chkRefreshAutomatically</zorder>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_refresh.png</normaloff>:/resource/16x16_ico_png/ico_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_filter.png</normaloff>:/resource/16x16_ico_png/ico_filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter ...</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_stop.png</normaloff>:/resource/16x16_ico_png/ico_stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_download.png</normaloff>:/resource/16x16_ico_png/ico_download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_browse.png</normaloff>:/resource/16x16_ico_png/ico_browse.png</iconset>
   </property>
   <property name="text">
    <string>Browse ...</string>
   </property>
  </action>
  <action name="actionShowNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionShowAudio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionShowVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionShowImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionShowText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionShowArchive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Archive</string>
   </property>
  </action>
  <action name="actionShowParity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </action>
  <action name="actionShowDocument">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Document</string>
   </property>
  </action>
  <action name="actionShowOther">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </action>
  <action name="actionShowBroken">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Broken</string>
   </property>
  </action>
  <action name="actionShowDeleted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deleted</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_delete.png</normaloff>:/resource/16x16_ico_png/ico_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionHeaders">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_headers.png</normaloff>:/resource/16x16_ico_png/ico_headers.png</iconset>
   </property>
   <property name="text">
    <string>View Blocks</string>
   </property>
   <property name="toolTip">
    <string>View data block details</string>
   </property>
  </action>
  <action name="actionBookmark">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_bookmark.png</normaloff>:/resource/16x16_ico_png/ico_bookmark.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Bookmark</string>
   </property>
   <property name="toolTip">
    <string>Toggle bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionBookmarkNext">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_bookmark_next.png</normaloff>:/resource/16x16_ico_png/ico_bookmark_next.png</iconset>
   </property>
   <property name="text">
    <string>Next Bookmark</string>
   </property>
   <property name="toolTip">
    <string>Find next bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionBookmarkPrev">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_bookmark_prev.png</normaloff>:/resource/16x16_ico_png/ico_bookmark_prev.png</iconset>
   </property>
   <property name="text">
    <string>Prev Bookmark</string>
   </property>
   <property name="toolTip">
    <string>Find previous bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>btnLoadMore</tabstop>
  <tabstop>chkRefreshAutomatically</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
